common.mk

```diff
@@ -17,25 +17,24 @@ endif
 script_env = \
 	SKIP_SQUASH=$(SKIP_SQUASH)                      \
 	UPDATE_BASE=$(UPDATE_BASE)                      \
-	VERSIONS="$(VERSIONS)"                          \
 	OS=$(OS)                                        \
-	VERSION="$(VERSION)"                            \
 	BASE_IMAGE_NAME=$(BASE_IMAGE_NAME)              \
 	OPENSHIFT_NAMESPACES="$(OPENSHIFT_NAMESPACES)"
 
 .PHONY: build
-build: manpages
-	$(script_env) $(build)
+build: $(VERSIONS)
```

Comment:
Correct, if the user has setup to run make in parallel by default. But I agree this is still an issue. 

The bug mentioned in  sclorg/postgresql-container#149 happens when testing, not building, so to work around this issue for the time being we could leave the build rule as-is and modify the test rule to not be able to be run in parallel (possibly by separating `build.sh` into building and testing parts)